### ISSUE TITLE
Unhandled match case 'xyz'

### ISSUE THREAD

**Issue by alxndrmlr** [livewire/flux#452]
Your call if you want this behaviour but I was getting tripped up with an `Unhandled match case 'gray'` when using 

```
<flux:badge color="gray">New</flux:badge>
```

Obviously this is to be expected when you look at the underlying code, where I got hit with it is because we have some code that was dynamically trying to set that color value and we didn't hit the error until we deployed.

Feels like it would be nicer to have a default fall back to `zinc` since that's the default value of the `prop`.

Its totally a taste thing though, it nipped me, we fixed it and carried on. Down side of setting the default is people start thinking they can send any color in here and then they don't realize they're not getting what they expected

**Comment by calebporzio** [comment]
Yeah, makes sense. I made invalid colors now default to zinc instead of error out.
